src/main/java/pl/wedding/management/model/employee/EmployeeRowFactory.java

```diff
@@ -0,0 +1,16 @@
+package pl.wedding.management.model.employee;
```

Comment:
RowFactory nie powininen się znajdować w katalogu `model`

Reply:
ale jak przerzucę go do innego folderu to będę musiał upublicznić entity